Makefile

```diff
@@ -382,6 +382,12 @@ endif
 	# On FreeBSD, remove the build's libdir from each library's RPATH
 ifeq ($(OS),FreeBSD)
 	$(JULIAHOME)/contrib/fixup-rpath.sh $(build_depsbindir)/patchelf $(DESTDIR)$(libdir) $(build_libdir)
+	$(JULIAHOME)/contrib/fixup-rpath.sh $(build_depsbindir)/patchelf $(DESTDIR)$(private_libdir) $(build_libdir)
+	$(JULIAHOME)/contrib/fixup-rpath.sh $(build_depsbindir)/patchelf $(DESTDIR)$(bindir) $(build_libdir)
+	# Set libgfortran's RPATH to ORIGIN instead of GCCPATH
+	for lib in $(DESTDIR)$(private_libdir)/libgfortran*$(SHLIB_EXT)*; do \
+		$(build_depsbindir)/patchelf --set-rpath '$$ORIGIN' $$lib; \
+	done
```

Comment:
Do we actually only need to do this for `libgfortran`, or do we need to do it for `libgcc_s` and friends as well?  If only `libgfortran`, I think we should add a more explicit note saying why we need to do this.

Reply:
It's just `libgfortran` in this case; of the GCC library clan, it's the only with its RPATH set. If we don't fix it, it fails to find `libgcc_s` and `libquadmath` on systems without the GCC port installed.

Reply:
Right; that makes sense.  Weird how it's the only one.  I would put your comment here into the comment in `Makefile`.